experimental/dds/attributable-map/src/test/types/validateMapPrevious.generated.ts

```diff
@@ -0,0 +1,231 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
```

Comment:
let's just turn type validation off--delete this file and remove all of the scripts in package.json which relate to type validation. that might be why PR build is failing as well.